### DISCUSSION TITLE
are there any plans to make an Apollo portable exe?

### DISCUSSION THREAD

**Discussion by yaole-A** [ClassicOldSong/Apollo#78] [General]
Do you have any plans to make a portable exe? I need multiple users, but I can only run one when I use multiple users. Sunshine can use portable exe to solve the problem of multiple users running two instances.

**Comment by ClassicOldSong** [comment]
It's well documented for how to start multiple instances in the Wiki: https://github.com/ClassicOldSong/Apollo/wiki/How-to-start-multiple-instances-of-Apollo
